_posts/2016-12-04-encore.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: ENCORE ensemble similarity
+---
+
+The **ENCORE** ensemble similarity library has been integrated in the next
+version of MDAnalysis
+as
+[MDAnalysis.analysis.encore](http://www.mdanalysis.org/mdanalysis/documentation_pages/analysis/encore.html).
+It implements a variety of techniques for calculating similarities between
+structural ensembles (trajectories), as described in this publication:
```

Comment:
Thanks for the work and the comments - I'm reviewing the blog post. @kain88-de, for the bootstrapping, guess you mean better docs in the docs of bootstrapping itself rather than the blog post?